src/App.js

```diff
@@ -1,35 +1,29 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
 import Wrapper from './layout/Wrapper';
 import Navbar from './components/navbar/Navbar';
 import Header from './layout/Header';
 import Footer from './layout/Footer';
-import Gym from './components/partner/Gym';
-import User from './components/partner/User';
-import Partner from './components/partner/Partner';
-import Signup from './components/signup/Signup';
-
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import Home from './components/Home';
+import Routers from './Routers';
```

Comment:
This is very nitpicky, but I would probably call this `Routes` not `Routers`. 
Router is something that "routes" incoming traffic to the correct "page"
While a route is a single "page"